lectures/markov_chains_II.md

```diff
@@ -196,16 +196,16 @@ This gives us another way to interpret the stationary distribution (provided irr
 
 Importantly, the result is valid for any choice of $\psi_0$.
 
-The theorem is related to {doc}`the Law of Large Numbers <lln_clt>`.
+The theorem is related to {doc}`the law of large numbers <lln_clt>`.
 
 It tells us that, in some settings, the law of large numbers sometimes holds even when the
-sequence of random variables is [not IID](iid_violation).
+sequence of random variables is {ref}`not IID  <iid_violation>`.
```

Comment:
@longye-tian , is this different?  I have a feeling the html will be the same...

Reply:
It will be the same. I changed this line while I was doing troubleshooting. (I was a bit desperate hhaha :))

Reply:
I just changed them back

Best ❤️ 
Longye